proposals/dao/fip_24.js

```diff
@@ -0,0 +1,75 @@
+/*
+ DAO Proposal Steps
+    1. Mint 1 million FEI to the optimistic timelock
+    2. Withdraw 50 ETH from old reserve stabilizer and send to OA multisig
+    3. Withdraw remaining eth in old reserve stabilizer and send to EthLidoPCVDeposit
+    4. Call deposit on the EthLidoPCVDeposit contract.
+*/
+
+/* Params:
+    addresses - all addresses in /contract-addresses/mainnetAddresses.json in an object of depth 1
+    oldContracts - an object of Web3 contract instances using the contract state from BEFORE any upgrades
+    contracts - an object of Web3 contract instances using the contract state from AFTER any upgrades
+        for example if a new UniswapPCVDeposit is deployed, the new instance is in contracts and the old instance is in oldContracts
+    logging - a flag for whether to log updates to the console
+*/    
+
+// Setup:
+// 1. Any protocol changes expected between current mainnet state and 
+//    fip execution on chain
+//    for example in `indexOTC` the counterparty needed to approve INDEX before the DAO vote could execute
+// 2. Any mocks needed to call run() (like impersonateAccount)
+async function setup(addresses, oldContracts, contracts, logging) {}
+
+// Runs the DAO proposal steps listed above
+// Note that these mock the on-chain proposal, but the actual execution will be handled by the governor
+async function run(addresses, oldContracts, contracts, logging) {
+    const {
+        fei,
+        tribalChiefOptimisticTimelock,
+        ethLidoPCVDeposit,
+        oldEthReserveStabilizer
+    } = contracts;
+
+    await fei.mint(tribalChiefOptimisticTimelock.address, '100000000000000000000');
```

Comment:
nit: elsewhere we define an `e18` variable with 18 zeros and use string interpolation to make these more readable

Reply:
Fixed